operator/src/main.rs

```diff
@@ -197,9 +240,10 @@ async fn main() -> Result<()> {
     info!("Confidential clusters operator",);
     let cl = Api::<ConfidentialCluster>::namespaced(client.clone(), &namespace);
 
-    tokio::spawn(install_trustee_configuration(client.clone(), namespace));
+    tokio::spawn(install_trustee_configuration(client.clone()));
+    tokio::spawn(install_register_server(client.clone()));
```

Comment:
For later, these 2 functions needs to be moved in the reconciliation loop if, for example, cocl cr doesn't exist yet